rs/registry/canister/src/mutations/do_swap_node_in_subnet_directly.rs

```diff
@@ -0,0 +1,163 @@
+use std::fmt::Display;
+
+use candid::CandidType;
+use ic_types::PrincipalId;
+use prost::Message;
+use serde::{Deserialize, Serialize};
+
+use crate::registry::Registry;
+
+/// Feature flag used to enable/disable the swapping feature on the whole network.
+const ENABLED: bool = false;
```

Comment:
You might use the Temporary library here (search in nns/sns) which allows you to enable/disable the feature flags in tests pretty easily.